openid-federation-1_0.xml

```diff
@@ -3418,14 +3418,35 @@
 	  </section>
         </section>
 
-        <section title="Validating a Trust Mark" anchor="trust_valid">
+        <section title="Validating a Trust Mark" anchor="trust-mark-validation">
           <t>
 	    An Entity SHOULD NOT try to validate a Trust Mark until
 	    it knows which Trust Anchor it is using.
-	    To validate a Trust Mark Issuer, follow the procedure
-	    defined in <xref target="resolving_trust"/>.
+	    To determine this, first resolve the Trust Chain,
+	    as described in <xref target="resolving_trust"/>.
+	    (TBD THEN DO WHAT?)
+	  </t>
+	  <t>
 	    To validate a Trust Mark, either:
+	    <list style="symbols">
+              <t>
+                Use the Trust Mark Status endpoint to verify that the
+                Trust Mark is still active
+                as described in <xref target="status_endpoint"/>.
+              </t>
+            </list>
+	    or perform these steps:
             <list style="symbols">
+	      <t>
+		Validate that the Trust Mark Issuer is part of the Federation
+		and that it is possible to get verified metadata about it
+		by using the procedure defined in <xref target="resolving_trust"/>.
+	      </t>
+	      <t>
+		If delegation is not being used (TBD HOW TO DETERMINE THIS?),
```

Comment:
Should be fine to check if the `delegation` claim is present in the trust mark.

However, the next points suggests to look at the TA's `trust_mark_owners` claim. So I woudl say there are two approaches:

- TA-centered: First checking if the TM is supposed to use delegation.
- TM-centered: First checking if the TM actually uses delegation.